shoop/configuration.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from __future__ import unicode_literals
+
+from shoop.core import cache
+from shoop.core.models import ConfigurationItem
+
+
+SHOP_CONFIGURATION_NAMESPACE = "shop_config"
+
+
+def _get_cache_key(shop):
+    return "%s:%s" % (SHOP_CONFIGURATION_NAMESPACE, shop.pk if shop else "")
+
+
+def set(shop, key, data):
+    """
+    Set configuration item
+
+    After configuration create or update invalidate cache
+
+    :param shop: shop for configuration
+    :type shop: shoop.core.models.Shop
+    :param key: configuration key
+    :type key: str
+    :param data: data set to configuration
+    :type data: JSON serializable object
+    """
```

Comment:
I had a better docstring written for this in previous rounds comments. Why not use it?